src/rust-2018/incremental-compilation-for-faster-compiles.md

```diff
@@ -0,0 +1 @@
+# Incremental Compilation for faster compiles
```

Comment:
Looks like an accidental page? This is included in incremental-compilation.md

Reply:
other way around; that one is accidental but yes